web/app/main.js

```diff
@@ -35,8 +36,10 @@ var selectedCountryCode;
 var forceRemoteEndpoint = false;
 var customDate;
 var timelineEnabled = false;
+var reqLang = 'en';
```

Comment:
how is the language changed except by querystring argument? Shouldn't it be detected automatically? Maybe you can use ~L134 where we set the `moment` locale?

```javascript
// Set proper locale
var locale = window.navigator.userLanguage || window.navigator.language;
moment.locale(locale);
```